io.neovim.nvim.tool.tmux.yml

```diff
@@ -0,0 +1,56 @@
+app-id: io.neovim.nvim.tool.tmux
+branch: stable
+runtime: io.neovim.nvim
+runtime-version: stable
+sdk: org.freedesktop.Sdk//22.08
+build-extension: true
+separate-locales: false
+appstream-compose: false
+
+modules:
+  - name: libevent
+    sources:
+      - type: archive
+        url: https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
+        sha256: 92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=${FLATPAK_DEST} --enable-shared
+      - make
+      - make install
+  - name: libutempter
+    sources:
+      - type: archive
+        url: https://github.com/altlinux/libutempter/archive/refs/tags/1.2.1-alt1.tar.gz 
+        sha256: 258d8bc4a42d2cbe4df6415912ab842574899adddbfc89d14ab56b57bb81158b 
+    buildsystem: simple
+    build-commands:
+      - make -C libutempter DESTDIR=${FLATPAK_DEST} libdir=/lib libexecdir=/lib includedir=/include
+      - make -C libutempter DESTDIR=${FLATPAK_DEST} libdir=/lib libexecdir=/lib includedir=/include install
```

Comment:
`FLATPAK_DEST` is not a `DESTDIR`. It's a `PREFIX`. DESTDIR is for staging.

Reply:
Fixed https://github.com/flathub/flathub/pull/4119/commits/bfa7933eeb36a770cd4868949cfd091cc67a24df